meta/main.yml

```diff
@@ -1,14 +1,14 @@
 ---
-dependencies:
-  - elliotweiser.osx-command-line-tools
-
 galaxy_info:
   author: geerlingguy
-  description: Homebrew for macOS
+  description: Homebrew for macOS and Debian
   company: "Midwestern Mac, LLC"
   license: "license (BSD, MIT)"
   min_ansible_version: 2.5
   platforms:
+    - name: Debian
```

Comment:
This should also work on `Ubuntu` the way it's currently structured. And with a couple tweaks mentioned elsewhere in this review, it should also work on `EL` (Red Hat / CentOS) and `Fedora`.

Reply:
I guess I assumed that the `platforms` in the meta file were the same as `ansible_os_family`.

See below.